pydiscourse/client.py

```diff
@@ -495,6 +519,8 @@ def category_topics(self, category_id, **kwargs):
             **kwargs
         )
 
+    # Doesn't work on recent Discourse versions (2014+)
+    # https://github.com/discourse/discourse_api/pull/204
     def hot_topics(self, **kwargs):
```

Comment:
If hot_topics doesn't work, perhaps we should remove the call. Alternatively I suppose it could return top_topics and return that rather than an error.
Thoughts?

Reply:
I don't know what is the strategy of `pydiscourse` for Discourse version support.
If a user needs support for an old Discourse version and wants to keep `pydiscourse` up-to-date, it may need this old method.
This is why I also did not updated the `category` function.

But if you want me to remove it, I can do it !

Reply:
5 years is a very old discourse instance, but I suppose they might still be hanging around - hopefully inside an organisation not on the web :)
Thanks for considering these legacy users !